charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -359,6 +359,7 @@ node:
   podAnnotations: {}
   podLabels: {}
   tolerateAllTaints: true
+  terminationGracePeriodSeconds: 30
```

Comment:
nit: swap places with `tolerateAllTaints` line to preserve pseudo-alphabetical order and not divide the two taint related parameters.

Reply:
Adressed with latest commit